webofneeds/won-core/src/main/java/won/protocol/util/linkeddata/CachingLinkedDataSource.java

```diff
@@ -242,9 +255,11 @@ private DatasetResponseWithStatusCodeAndHeaders fetchOnlyOnce(final URI resource
         CountDownLatch preExistingLatch = countDownLatchMap.putIfAbsent(cacheKey, latch);
         try {
             if (preExistingLatch != null) {
-                logger.debug("resource " + cacheKey
-                                + " is being fetched in another thread, we wait for its result and use it "
-                                + "if it turns out to be cacheable");
+                if (logger.isDebugEnabled()) {
+                    logger.debug("resource " + cacheKey
```

Comment:
not to nitpick, but if you use a parameterized query ( .. {}... with the cacheKey as a parameter) you could avoid the isDebugEnabled clause, because the logger framework would only concat/init the log-string if the debug mode is enabled anyway

Reply:
fixed, thanks